name change et al

doktordirk opened the pull request.

RWOverdijk commented on README.md:
```diff
@@ -17,14 +17,20 @@ This library is used directly by applications only.
 
 Either set your own styles or install humane.js with `jspm install github:wavded/humane-js` to have some default styles to choose from.
 
+Npm installation for eg webpack is currently not possible.
```
Isn't it? I thought this PR would fix that. Surely it's possible if you bundle the deps?

doktordirk replied:
github thingy. might be simple fix though

doktordirk replied:
i'll get it fixed. (needs update of  i18n for this)

doktordirk committed:
chore(project): remove unneeded babel devDependency

doktordirk committed:
feat(project): export Config

doktordirk committed:
feat(project): Rename project to remove spoonx prefix

doktordirk committed:
feat(project): allow npm installation. update aurelia-i18n to ^ 0.5.0

BREAKING CHANGE: `wavded/` prefix dropped from install name humane-js. Update `package.json` and `config.js` accordingly.
